### PR TITLE
Happy birthday service

### PR DESCRIPTION
**Pull request by zsolt-molnar-heits**

### CODE REVIEW

File: app.yaml [hunk 1]
```diff
@@ -6,5 +6,10 @@ env_variables:
   SLACK_VERIFICATION_TOKEN: slack-verification-token
   WATERCOOLER_CHANNEL_ID: C0403NFNUJE
   GENERAL_CHANNEL_ID: CEC0Z16QL
+  BIRTHDAY_CHANNEL_ID: C04P503PXLY
```
Comment (zsolt-molnar-heits) [inline app.yaml:9]: this will be the general channel, but for testing reason I have created a dummy channel

File: cron.yaml [hunk 2]
```diff
@@ -6,3 +6,10 @@ cron:
     retry_parameters:
       min_backoff_seconds: 2.5
       max_doublings: 5
+  - description: 'happy birthday message'
+    url: /cron/birthday
+    target: default
+    schedule: every 15 minutes from 08:00 to 23:00
```
Comment (zsolt-molnar-heits) [inline cron.yaml:12]: this will be on a daily basis, current config is for testing session

File: internal/service/google_api.go [hunk 3]
```diff
@@ -0,0 +1,136 @@
...
+	entry := new(calendar.CalendarListEntry)
+	entry.Id = calendarId
+
+	googleCalendar.service.CalendarList.Insert(entry)
+
+	return entry
+}
+
+func GenerateBirthdayWish(description string) string {
+	expression, _ := regexp.Compile("\\<@(.*?)\\>")
```
Comment (iulian-balan) [inline internal/service/google_api.go:76]: this seems complicated and prone to miss something when inserting data. 
We could use the email address and slack api to get the slack user id.

**Review by iulian-balan — APPROVED** [review]

File: internal/service/google_api.go (hunk 3 above)
Comment (zsolt-molnar-heits) [reply internal/service/google_api.go:76]: yup we should do this in an other iteration, I would like to move forward with prod testing. HR was instructed how to introduce properly the data using the userid

**Pull request merged**